ibis/tests/expr/test_string.py

```diff
@@ -96,16 +95,6 @@ def test_contains(table):
         "foo" in table.g  # noqa: B015
 
 
-@pytest.mark.parametrize(
-    ("left_slice", "right_start", "right_stop"),
-    [(slice(None, 3), 0, 3), (slice(2, 6), 2, 4)],
-)
-def test_getitem_slice(table, left_slice, right_start, right_stop):
```

Comment:
this test is checking for an implementation detail, it probably should never have existed.

Reply:
Agreed.